demo/test.c

```diff
@@ -564,6 +612,28 @@ static int test_mp_invmod(void)
       EXPECT(mp_cmp(&c, &d) == MP_EQ);
    }
 
+   /* Some small general tests https://github.com/libtom/libtommath/issues/534 */
+   for (i = -10; i < 11; i++) {
+      for (j = -10; j < 11; j++) {
+         mp_set_i32(&a, i);
+         mp_set_i32(&b, j);
+         e = mp_invmod(&a, &b, &c);
+         if (e != MP_OKAY) {
+            if (results[i+10][j+10] != -1) {
+               printf("error = %s from ", mp_error_to_string(e));
+               printf("error at i = %d, j =%d should be an error but gave %d\n",i,j, results[i+10][j+10]);
```

Comment:
Shouldn't this print out the value of `c` instead of the expected results table?

Reply:
Yes, of course it should!
Thanks!